lua/tardis/parts/legacy/legacy_physbrake.lua

```diff
@@ -0,0 +1,21 @@
+--Legacy Interior - Physics Lock
+
+local PART = {}
+PART.ID = "legacy_physbrake"
+PART.Name = "Legacy Physics Lock"
+PART.Model = "models/drmatt/tardis/handbrake.mdl"
+PART.AutoSetup = true
+PART.Collision = true
+PART.Animate = true
+
+if SERVER then
+    function PART:Use(ply)
+        self:EmitSound("tardis/control_handbrake.wav")
+        if self.InSequence==true then return end
```

Comment:
Should this be handled by the sequence system itself? Individual parts shouldn't have to care about the sequence system. Maybe should define PART.Sound and call that from the higher code which calls PART:Use? Open to suggestions here just don't want a bunch of duplicated sequence checking code

Reply:
Good point, I'll see to that